dtfs-central-api/src/services/state-machines/fee-record/event-handlers/generate-keying-data/generate-keying-data.event-handler.test.ts

```diff
@@ -32,121 +33,224 @@ describe('handleFeeRecordGenerateKeyingDataEvent', () => {
   const aMatchingFeeRecord = () => FeeRecordEntityMockBuilder.forReport(aReconciliationInProgressReport()).withStatus('MATCH').build();
 
   beforeEach(() => {
-    jest.resetAllMocks();
     jest.mocked(calculateFixedFeeAdjustment).mockResolvedValue(10);
```

Comment:
Is this line not redundant as the last mock implementation takes precedence?

Reply:
Ah whoops yes, the line below should be calculatePrincipleBalanceAdjustment! Will update now